lib/api/storage.ts

```diff
@@ -18,11 +20,15 @@ export class ShredderStorage {
   }
 
   static getRaw() {
-    return localStorage.getItem(`${ location.origin }-shredder`);
+    return localStorage.getItem(this.storageKey);
   }
 
   static getSerialized() {
-    return JSON.stringify(ShredderStorage.getRaw());
+    return JSON.stringify(this.getRaw());
+  }
+
+  static getParsed() {
+    return JSON.parse(this.getRaw());
```

Comment:
This is a static method, we are not using `this` here as a best practice to avoid confusion with instance methods.